lib/storage/src/types.rs

```diff
@@ -23,3 +24,47 @@ pub struct StorageConfig {
     pub performance: PerformanceConfig,
     pub hnsw_index: HnswConfig,
 }
+
+/// Information of a peer in the cluster
+#[derive(Debug, Deserialize, Serialize, JsonSchema, Clone)]
+pub struct PeerInfo {
+    pub uri: String,
+    // ToDo: How long ago was the last communication? In milliseconds
+    // pub last_responded_millis: usize
+}
+
+/// Summary information about the current raft state
+#[derive(Debug, Deserialize, Serialize, JsonSchema, Clone)]
+pub struct RaftInfo {
+    /// Raft divides time into terms of arbitrary length, each beginning with an election.
+    /// If a candidate wins the election, it remains the leader for the rest of the term.
+    /// The term number increases monotonically.
+    /// Each server stores the current term number which is also exchanged in every communication.
+    pub term: u64,
+    // ToDo: What does this mean?
+    pub commit: u64,
```

Comment:
The index of the latest committed (finalized) operation that this peer is aware of.